src/register/RegistrationPage.jsx

```diff
@@ -216,18 +208,31 @@ class RegistrationPage extends React.Component {
       honor_code: true,
       country: this.state.country,
     };
+    const { name, value } = e.target;
     this.setState({
-      assignRegistrationErrorsToField: false,
+      updateFieldErrors: false,
+      updateAlertErrors: false,
+    }, () => {
+      this.validateInput(name, value, payload, false);
     });
+  }
 
-    this.props.fetchRealtimeValidations(payload);
+  handleOnChange(e) {
```

Comment:
handleOnChange is not used any where is code. Need to update the onChange handler for `AuthnValidationFormGroup`